recipe/meta.yaml

```
@@ -1,39 +1,46 @@
 {% set name = "textstat" %}
-{% set version = "0.7.2" %}
+{% set version = "0.7.3" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: d04aa648ffbfb54ee5ab4b3d434575c83a69b45704e6a3cbc03344d17552e819
+  sha256: 60b63cf8949f45bbb3b4205e4411bbc1cd66df4c08aef12545811c7e6e24f011
 
 build:
-  noarch: python
   number: 0
-  script: {{ PYTHON }} -m pip install . -vv
+  skip: True  # [py<36]
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
 
 requirements:
   host:
     - pip
-    - python >=3.6
+    - python
+    - setuptools
+    - wheel
   run:
     - pyphen
-    - python >=3.6
-    - setuptools
+    - python
 
 test:
   imports:
     - textstat
+  requires:
+    - pip
+  commands:
+    - pip check
```

we could add tests. I did not execute this but it should work:

```suggestion
  requires:
    - pip
    - pytest
  source_files:
    - test.py
  commands:
    - pip check
    - pytest test.py
```

---

yes, that is good idea

---

`test.py` isn't available inside the Source Distribution https://pypi.org/project/textstat/0.7.3/#files

---

😞